app/models/request_type/validator.rb

```diff
@@ -70,10 +70,14 @@ def validate?(_value)
     def default
       nil
     end
+
+    def valid_options
+      []
+    end
   end
```

Comment:
Mostly to help cases in the test suite where we didn't have library types associated with a request type. It would fall back to the permissive 'NullValidator' but in some cases would try and get a list of options for drop-downs etc. We might need to revisit this is its something that starts happening in production (where it might make more sense to find *all* library types) but currently this is more of a theoretical scenario, so we just don't suggest any options.